assets/js/blocks/attribute-filter/block.tsx

```diff
@@ -666,7 +666,7 @@ const AttributeFilterBlock = ( {
 								),
 							} }
 						/>
-						{ multiple && (
+						{ checked.length === 0 && ! isLoading && (
```

Comment:
That change covers the case where dropdown is set to Single choice, but it little bug for dropdowns with Multiple choice: when user selected at least one option, the chevron would be hidden, even though there's more options to choose from. So this case should also be covered. Maybe something among those lines:

```suggestion
						{ multiple || checked.length === 0 && ! isLoading && (
```

There's also [this issue](https://github.com/woocommerce/woocommerce-blocks/issues/7889) about hiding chevron if there's no more options to choose from in Multiple choice dropdowns, which now I think it's much easier than I thought at the time I created the issue 🙈 I think the above suggestion could be extended with:

```
const showChevron = multiple ?
  checked.length < displayedOptions.length : // MULTIPLE: show chevron only when not every option is chosen
  checked.length === 0                       // SINGLE: show chevron only when none option is chosen
```

to cover each case.

But if you feel that's out of scope of this PR it's fine and you can just stick to the first suggestion 🙌